### PR TITLE
Prepare 0.3.0 release

### PR DESCRIPTION
**Pull request by rentziass**
Notable Changes (excluding test/doc):

- #2348
- #2268
- #2381
- #2366
- #2279
- #2341
- #2347

### CODE REVIEW

**Review by nikola-jokic — DISMISSED** [review]
LGTM

File: docs/preview/gha-runner-scale-set-controller/README.md [hunk 1]
```diff
@@ -158,6 +158,18 @@ Verify that the secret you provided is correct and that the `githubConfigUrl` yo
 
 ## Changelog
 
+### v0.3.0
+
+#### Major changes
+
+1. Runner pods are more similar to hosted runners (#2348)
+1. Add support for self-signed CA certificates (#2268)
+1. Trailing slashes in config URLs don't break installations (#2381)
```
Comment (Link-) [inline docs/preview/gha-runner-scale-set-controller/README.md]:
```suggestion
1. Fixed trailing slashes in config URLs breaking installations [#2381](https://github.com/actions/actions-runner-controller/pull/2381)
```

File: docs/preview/gha-runner-scale-set-controller/README.md [hunk 2]
```diff
@@ -158,6 +158,18 @@ Verify that the secret you provided is correct and that the `githubConfigUrl` yo
...
+1. Fix a bug where the listener pod would ignore proxy settings from env (#2366)
```
Comment (Link-) [inline docs/preview/gha-runner-scale-set-controller/README.md]:
```suggestion
1. Fixed a bug where the listener pod would ignore proxy settings from env [#2366](https://github.com/actions/actions-runner-controller/pull/2366)
```

File: docs/preview/gha-runner-scale-set-controller/README.md [hunk 3]
```diff
@@ -158,6 +158,18 @@ Verify that the secret you provided is correct and that the `githubConfigUrl` yo
 
 ## Changelog
 
+### v0.3.0
+
+#### Major changes
+
+1. Runner pods are more similar to hosted runners (#2348)
```
Comment (Link-) [inline docs/preview/gha-runner-scale-set-controller/README.md]: Sadly, GitHub doesn't render these URLs in markdown. So you have manually add them

Comment (Link-) [inline docs/preview/gha-runner-scale-set-controller/README.md]:
```suggestion
1. Runner pods are more similar to hosted runners [#2348](https://github.com/actions/actions-runner-controller/pull/2348)
```

File: docs/preview/gha-runner-scale-set-controller/README.md [hunk 4]
```diff
@@ -158,6 +158,18 @@ Verify that the secret you provided is correct and that the `githubConfigUrl` yo
...
+1. Add support for self-signed CA certificates (#2268)
+1. Trailing slashes in config URLs don't break installations (#2381)
+1. Fix a bug where the listener pod would ignore proxy settings from env (#2366)
+1. Runner set name is now configurable (#2279)
+1. Name and namespace labels of listener pod have been split (#2341)
+1. Chart naming validation on AutoscalingRunnerSet install (#2347)
```
Comment (Link-) [inline docs/preview/gha-runner-scale-set-controller/README.md]:
```suggestion
1. Added chart name constraints validation on AutoscalingRunnerSet install [#2347](https://github.com/actions/actions-runner-controller/pull/2347)
```

File: docs/preview/gha-runner-scale-set-controller/README.md [hunk 5]
```diff
@@ -158,6 +158,18 @@ Verify that the secret you provided is correct and that the `githubConfigUrl` yo
...
 
+### v0.3.0
+
+#### Major changes
+
+1. Runner pods are more similar to hosted runners (#2348)
+1. Add support for self-signed CA certificates (#2268)
+1. Trailing slashes in config URLs don't break installations (#2381)
+1. Fix a bug where the listener pod would ignore proxy settings from env (#2366)
+1. Runner set name is now configurable (#2279)
```
Comment (Link-) [inline docs/preview/gha-runner-scale-set-controller/README.md]:
```suggestion
1. Added runner set name field making it optionally configurable [#2279](https://github.com/actions/actions-runner-controller/pull/2279)
```

**Review by Link- — CHANGES_REQUESTED** [review]
We need to fix the change log formatting

**Review by nikola-jokic — APPROVED** [review]

**Review by Link- — APPROVED** [review]
🚀

**Pull request merged**